lib/server.js

```diff
@@ -11,14 +10,16 @@ module.exports = (args, callback) => {
     return callback(new Error('Email is required for authorization. Please set your email in _config.yml.'));
   }
 
-  const webuiConfig = config.webui = config.webui || {};
-
   // Default web UI config
-  _.defaults(webuiConfig, {
+  let webuiConfig = Object.assign({}, {
     password: util.randomString(8, {uppercase: false}),
     port: 5000
   });
 
+  if (typeof config.webui !== 'undefined') {
+    webuiConfig = Object.assign(webuiConfig, config.webui);
+  }
+
   const app = express();
```

Comment:
`let webuiConfig` to `const webuiConfig` and `webuiConfig = Object.assign(webuiConfig, config.webui)` to `Object.assign(webuiConfig, config.webui);`
The first argument of `Object.assign()` is the object to which the property is assigned.
Therefore, there is no need to reassign this line to `webuiConfig`.
Therefore, `webuiConfig` can be declared by const statement.

Reply:
Addressed. I also refactored to `if (config.webui)` since `config` is always defined.